### ISSUE TITLE
Users can show up multiple times if they get cut off and reconnect

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#3601]
*Issue migrated from trac ticket [#4891](http://colloquy.info/project/ticket/4891)*

**type:** Defect | **component:** Colloquy (iOS) | **severity:** Normal | **version:** 1.8 (iOS) | **status:** new | **keywords:** Userlist, names, nicknames,
___
#### 2016-04-16 16:45:03: anonymous created the issue

If a user is disconnected while I'm chatting to them privately there is no message saying they are offline. When they return, their name appears twice in the user list. If they have repeated d/c problems, their name can appear multiple times. It'd be useful if Colloquy could detect when they go offline to avoid my inputting messages that the other person doesn't see.
___
#### 2016-04-22 11:37:44: Aethelstan <niceguyfun@… commented

To sum this up - Users aren't being removed from the list when they log off, so you end up with multiple names on occasions and if you've not noticed that a user has logged off you can end up messaging people who aren't there which is annoying!
So we have users showing online when they're offline, and we still have the problem of the app reporting that a user is offline when they're online!
In addition the 'idle' time counter still isn't working correctly.

**Comment by zadr** [comment]
this should be fixed in 1.8.11, coming in the next few days
